src/prx/helpers.py

```diff
@@ -362,10 +368,16 @@ def cached_load(rinex_file: Path, file_hash: str):
 
 def get_gpst_utc_leap_seconds_from_rinex_header(rinex_file: Path):
```

Comment:
The function name should probably change, as the leap second is not always coming from the header now. Something `get_gpst_utc_leap_seconds` would work.

Reply:
modified in #81, 9b52d2560987b9f449ec7a535e6a3fc461822526